Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -51,7 +51,7 @@ function heading() {
 }
 
 # Install lsb-release as it is needed to check Ubuntu version
-if package_is_installed "lsb-release" -eq 1; then
+if package_is_installed "lsb-release" -eq 0; then
```

Comment:
this is weird, it looks like package_is_installed uses exit codes, not returned integers, so why does this work?

Reply:
Corrected !